c2rust-refactor/scripts/reconstruct_for_range.lua

```diff
@@ -37,7 +37,7 @@ $'label:?Ident: while $cond:Expr {
                   print(incr)
                   if (incr.kind == "Semi" or
                       incr.kind == "Expr") then
-                     incr = incr.expr
+                     incr = incr.expr_old
```

Comment:
Can we get rid of `expr_old` and use the new style here?

Reply:
Yes - I suppose I didn't want to touch this at the time since I didn't know how the script works and since it expects an interned node, I wasn't sure how to fix it. But I can take the time to look at it I suppose